cmd/download.go

```diff
@@ -349,6 +362,43 @@ func (sf solutionFile) relativePath() string {
 	return filepath.FromSlash(file)
 }
 
+func trackAndSlugFromCwd(workspace string) (string, string, bool) {
```

Comment:
surely, it would be possible to call this `flagsFromCwd` and have it automatically detect the `team` if inside a teams directory (from CWD) and the UUID if inside `users` (but this requires reading a file in order to get the UUID). Since I don't use either of these pieces of functionality, and the latter needs more thought, I would not like to take on such a task in this PR.